user/ani-cli/template.py

```diff
@@ -0,0 +1,17 @@
+pkgname = "ani-cli"
+pkgver = "4.9"
+pkgrel = 0
+depends = ["aria2", "chimerautils", "curl", "ffmpeg", "fzf", "mpv"]
```

Comment:
```suggestion
depends = ["aria2", "curl", "ffmpeg", "fzf", "mpv"]
```

always there

also isn't yt-dlp prefered to 'ffmpeg' given the readme or nah

Reply:
Good catch, never noticed it. Should be yt-dtp yeah.